packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -167,6 +175,23 @@ <h2>
                             </oui-select>
                         </oui-field>
                     </oui-inline-adder-field>
+                    <oui-inline-adder-field>
+                        <oui-field
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_prefix' | translate  }}"
+                            data-label-popover="{{:: 'pci_projects_project_training_jobs_submit_data_prefix_help' | translate }}"
+                        >
+                            <input
+                                type="text"
+                                class="oui-input"
+                                placeholder=""
+                                data-ng-init=""
+                                name="prefix"
+                                data-ng-model="prefix"
```

Comment:
```suggestion
                                data-ng-model="$ctrl.prefix"
```
If not `model` will be declared as part of the `$scope`

Reply:
fixed